lib/models/candidate.py

```diff
@@ -0,0 +1,91 @@
+from dataclasses import dataclass
+from typing import Optional
+
+from requests import HTTPError
+
+from lib.api.checkr_api import CheckrAPI
+from lib.helpers.checkr_api_error import CheckerAPIFailedCreation
+
+
+@dataclass
+class Candidate:
+    checkr_api: CheckrAPI
+    first_name: str
+    last_name: str
+    email: str
+    driver_license_number: str
+    driver_license_state: str
+    work_locations: list[dict[str, str]]
+
+    id: Optional[str] = None
+    middle_name: Optional[str] = None
+    no_middle_name: Optional[bool] = None
+    mother_maiden_name: Optional[str] = None
+    ssn: Optional[str] = None
+    dob: Optional[str] = None
+    object: Optional[str] = None
+    zipcode: Optional[str] = None
+    uri: Optional[str] = None
+    phone: Optional[str] = None
+    geo_ids: Optional[list[str]] = None
+    previous_driver_license_number: Optional[str] = None
+    previous_driver_license_state: Optional[str] = None
+    copy_requested: Optional[bool] = None
+    custom_id: Optional[str] = None
+    report_ids: Optional[list[str]] = None
```

Comment:
Here and elsewhere: we're sure this is `list[str]` and not `list[int]`?

Reply:
Yes. I've checked with their [docs](https://docs.checkr.com/#tag/Candidates). I think all the ids are strings. 
![image](https://user-images.githubusercontent.com/63125562/200095412-9a7906de-8893-4b74-98b4-b7f976185b5d.png)